### PR TITLE
[Notifier] Add Engagespot bridge

### PR DESCRIPTION
**Pull request by danut007ro**
Add documentation about engagespot integration

https://github.com/symfony/symfony/pull/44303

### CODE REVIEW

File: notifier.rst [hunk 1]
```diff
@@ -331,13 +331,14 @@ integration with these push services:
 Service         Package                               DSN
 ==============  ====================================  =================================================================================
 Firebase        ``symfony/firebase-notifier``          ``firebase://USERNAME:PASSWORD@default``
+Engagespot      ``symfony/engagespot-notifier``        ``engagespot://API_KEY@default?campaign_name=DEFAULT_CAMPAIGN``
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
Engagespot      ``symfony/engagespot-notifier``        ``engagespot://API_KEY@default?campaign_name=CAMPAIGN_NAME``
```

File: notifier.rst [hunk 2]
```diff
@@ -331,13 +331,14 @@ integration with these push services:
...
 Expo            ``symfony/expo-notifier``              ``expo://Token@default``
 OneSignal       ``symfony/one-signal-notifier``        ``onesignal://APP_ID:API_KEY@default?defaultRecipientId=DEFAULT_RECIPIENT_ID''``
 ==============  ====================================  =================================================================================
 
 .. versionadded:: 5.4
 
-    The Expo and OneSignal integrations were introduced in Symfony 5.4.
+    The Engagespot, Expo and OneSignal integrations were introduced in Symfony 5.4.
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
    The Expo, OneSignal and Engagespot integrations were introduced in Symfony 5.4.
```

**Review by OskarStark — CHANGES_REQUESTED** [review]

**Comment by OskarStark** [comment]
Please retarget `6.1` branch

**Comment by danut007ro** [comment]
Done, but tests are failing, seems unrelated

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Can you please rebase? This should make the CI green. Thanks

**Comment by javiereguiluz** [comment]
@danut007ro sorry it took us so long to take care of your contribution. I was going to merge it, but there was lots of conflicts while merging. I couldn't fix those, so I recreated your changes directly in https://github.com/symfony/symfony-docs/commit/e16e05ab02983f166296564b4cb2a9b9b1c33db2 Thanks!

**Pull request closed without merging**